pymeasure/display/widgets/image_widget.py

```diff
@@ -113,3 +150,9 @@ def load(self, curve):
 
     def remove(self, curve):
         self.plot.removeItem(curve)
+
+    def _update_colorbar(self):
+        """Updates the colorbar to match the current image"""
+        for item in self.plot.items:
+            if isinstance(item, ResultsImage):
+                self.colorbar.setImageItem(item, insert_in=self.plot)
```

Comment:
I think you can set multiple images at once to have a better link:
```suggestion
        items = [item for item in self.plot.items if isinstance(item, ResultsImage)]
        self.colorbar.setImageItem(items, insert_in=self.plot)
```

This way, the colorbar always matches with all of the figures. However the scale is auto-updated with the latest one (so the one that is currently running for example... I guess that is a limitation of leaving the auoleveling to the setImage function?